tests/rest/client/v1/test_presence.py

```diff
@@ -33,13 +33,13 @@ class PresenceTestCase(unittest.HomeserverTestCase):
 
     def make_homeserver(self, reactor, clock):
 
+        ph = Mock()
```

Comment:
Please don't use short variables like this, `presence_handler` is much more readable.